dashboard/modules/runtime_env/runtime_env_agent.py

```diff
@@ -289,32 +294,12 @@ async def _setup_runtime_env(
                 runtime_env, context, logger=per_job_logger
             )
 
-            for manager in self._base_plugin_cache_managers.values():
+            for manager in self._plugin_cache_managers.values():
```

Comment:
I'm sorry that you should refactor this because my PR about priority has been merged. We should consider the setup order here.

Reply:
Not a problem!  I'll fix the merge conflicts